ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml

```diff
@@ -229,6 +229,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi
+- as: e2e-metal-ovn-sno-cert-rotation-shutdown
+  optional: true
+  steps:
+    cluster_profile: packet-sno
+    env:
+      CLUSTER_AGE_DAYS: "1800"
+      CLUSTER_AGE_STEP: "300"
```

Comment:
Lets add
```
PACKET_OS=rocky_9
```
to every test so that it would pass on 4.16+

Reply:
don't you want to add this to the workflow itself?

Reply:
This would break 4.15 tests, so lets have it here. We'll probably drop 4.15 soon, but until then lets have them set explicitly